### PR TITLE
Add hv.reverb. Simple reverberator using recirculating delays

### PR DESCRIPTION
**Pull request by Simon-L**
This PR adds a cheap and simple reverb adapted from rjlib's reverb, see here: https://github.com/rjdj/rjlib/blob/master/rj/e_reverb.pd
And here for DSP details: http://msp.ucsd.edu/techniques/v0.11/book-html/node111.html

Test patch, including a JS demo: https://enzienaudio.com/h/FergusL/reverb/
And a cool filter using it: http://www.woldhek.eu/anton/?p=881

### CODE REVIEW

File: hv.reverb.pd [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+#X obj 35 317 * 0.125;
+#X obj 150 318 r \$0-wet-gain;
+#X obj 338 83 hv.dispatch \$0 wet-gain 80;
+#X obj 338 62 hv.dispatch \$0 dry-gain 20;
+#X obj 338 149 hv.dispatch \$0 crossfreq 3000;
+#X obj 178 362 r \$0-crossfreq;
+#X obj 338 41 hv.dispatch \$0 feedback 90;
+#X obj 136 50 delwrite~ \$0-predelay 3000;
+#X obj 136 136 sig~;
+#X obj 136 95 hsl 128 15 0 2800 0 0 empty empty empty -2 -8 0 10 -262144
```
Comment (diplojocus) [inline hv.reverb.pd]: Probably best to remove this

File: hv.reverb.pd [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+-1 -1 0 1;
+#X floatatom 136 115 5 0 0 0 - - -, f 5;
```
Comment (diplojocus) [inline hv.reverb.pd]: And this

**Comment by diplojocus** [comment]
Just a small comment about removing GUI elements from the main object, but otherwise looks great!

**Comment by Simon-L** [comment]
Indeed! Fixed.

File: README.md [hunk 3]
```diff
@@ -30,7 +30,9 @@ Processing
 
 `[hv.flanger2]` - Stereo version of `[hv.flanger]`.
 
-`[hv.freqshift]` - Frequency shifter effect unity.
```
Comment (diplojocus) [inline README.md:33]: Oh yeah! Thanks for the typo fix!

**Comment by diplojocus** [comment]
You get :1st_place_medal:

**Pull request merged**